src/gql/GQLWrapper.tsx

```diff
@@ -85,6 +86,18 @@ const authenticateIfSuccessfulLink = (dispatch: Dispatch) =>
     });
   });
 
+const retryLink = new RetryLink({
+  delay: {
+    initial: 300,
+    max: Infinity,
```

Comment:
This shouldn't come up at all since there's a max attempts, but you should set this to something finite